pkg/component/extensions/operatingsystemconfig/original/components/sshdensurer/component_test.go

```diff
@@ -124,18 +125,14 @@ fi
 	disableScript = `#!/bin/bash -eu
 set -e
 
-# Disable sshd service if enabled
-if systemctl is-enabled --quiet sshd.service ; then
-    systemctl disable sshd.service
+# Disable and mask sshd service if not masked
+if [ "$(systemctl is-enabled sshd.service)" != "masked" ] ; then
+    systemctl disable sshd.service || true
```

Comment:
Masking the service should be enough, so no need to call `systemctl disable ...` here.

Reply:
I agree. However the disablement of `SSHAccess` was made with the following DISA K8s STIGs rules in mind: [242393](https://www.stigviewer.com/stig/kubernetes/2023-08-29/finding/V-242393), [242394](https://www.stigviewer.com/stig/kubernetes/2023-08-29/finding/V-242394). In 242394 it is specified to disable the `sshd` unit.

I also find it a nice touch to know that even if the unit gets unmasked it would still be disabled.